spec/models/recipe_spec.rb

```diff
@@ -12,4 +23,8 @@
     it {should have_many :recipe_ingredients}
     it {should have_many(:ingredients).through(:recipe_ingredients)}
   end
+
+  it '#total_cost' do
+    expect(@spaghetti_bolognese.total_cost).to eq 4
+  end
```

Comment:
```ruby
def total_cost
  4
end
```

Does your assertion disprove that this is what your method looks like? Probably not. My suggestion, for instance methods, is to test more than 1 instance object so that way your code proves that the value isn't hard-coded and is dynamic.

```ruby
expect(@spaghetti_bolognese.total_cost).to eq 4
expect(@tacos.total_cost).to eq 3 # create another recipe with different ingredients
```